save-backend/build.gradle.kts

```diff
@@ -8,13 +8,18 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
     kotlin("jvm")
     // this plugin will generate generateOpenApiDocs task
-    // running this task, it will writes the OpenAPI spec into a backend-api-docs.json file in save-backend dir.
+    // running this task, it will write the OpenAPI spec into a backend-api-docs.json file in save-backend dir.
     id("org.springdoc.openapi-gradle-plugin") version "1.3.4"
 }
 
 openApi {
     apiDocsUrl.set("http://localhost:5800/internal/v3/api-docs/latest")
+    outputDir.set(file(rootDir))
     outputFileName.set("$rootDir/save-backend/backend-api-docs.json")
+    waitTimeInSeconds.set(120)
+
+    val bootRun = project.tasks["bootRun"] as org.springframework.boot.gradle.tasks.run.BootRun
```

Comment:
Shouldn't it be `forkedBootRun`? Also, it's preferrable to use `tasks.named` instead of `tasks.get`

Reply:
In current version of plugin there is no `forkedProperties`, the readme of this plugin is outdated

Changed to `tasks.named`

Reply:
Should be `project.tasks`, not `rootProject.tasks`

Reply:
Yeah, thanks